packages/ds-healthcare-gov/docs/src/pages/patterns/Footer/Footer.example.jsx

```diff
@@ -45,12 +45,27 @@ class FooterExample extends React.PureComponent {
   }
 
   render() {
-    return (
-      <div>
-        <Footer initialLanguage={this.locale} />
-        {this.renderFooterToggles()}
+    const wrapperClassNames = 'ds-u-padding--1';
+    const footerTopExample = (
+      <div className="ds-l-container ds-u-padding-bottom--2">
+        <div className="ds-u-text-align--right ds-u-border-bottom--1 ds-u-padding-bottom--2 ds-u-padding-right--2">
+          <span className="ds-u-font-size--xl ds-u-sans ds-u-font-weight--bold ds-u-color--gray-dark">
+            58 DAYS
+          </span>
+        </div>
```

Comment:
Super small nit pick, we could probably simplify this down to fewer nested elements.